great_expectations/core/expectation_configuration.py

```diff
@@ -1230,28 +1230,19 @@ def to_json_dict(self):
         return myself
 
     def get_evaluation_parameter_dependencies(self):
-        parsed_dependencies = {}
-        for key, value in self.kwargs.items():
-            if isinstance(value, dict) and "$PARAMETER" in value:
-                param_string_dependencies = find_evaluation_parameter_dependencies(
-                    value["$PARAMETER"]
-                )
-                nested_update(parsed_dependencies, param_string_dependencies)
-
+        parsed_dependencies = self._traverse_expectation_configuration_kwargs(
+            root=self.kwargs
+        )
         dependencies = {}
         urns = parsed_dependencies.get("urns", [])
         for string_urn in urns:
             try:
                 urn = ge_urn.parseString(string_urn)
             except ParserError:
-                logger.warning(
-                    "Unable to parse great_expectations urn {}".format(
-                        value["$PARAMETER"]
-                    )
-                )
+                logger.warning(f"Unable to parse great_expectations urn {string_urn}")
                 continue
 
-            if not urn.get("metric_kwargs"):
+            if "metric_kwargs" not in urn:
```

Comment:
Purely stylistic but I'd rather not use `get()` if we don't do anything with the result.